qiskit_experiments/framework/base_analysis.py

```diff
@@ -88,7 +88,9 @@ def copy(self) -> "BaseAnalysis":
 
     @classmethod
     def _default_options(cls) -> Options:
-        return Options()
+        options = Options()
+        options.figure_names = None
```

Comment:
This is a user facing option, so please add a short comment describing what this is used for and how